lib/stringify.js

```diff
@@ -74,7 +75,8 @@ var stringify = function stringify(
     formatter,
     encodeValuesOnly,
     charset,
-    sideChannel
+    sideChannel,
+    allowEmptyArrays
```

Comment:
let's insert this somewhere higher up, so sideChannel remains last, and so it's near something sensible